.github/actions/deploy/action.yml

```diff
@@ -13,8 +13,6 @@ runs:
       shell: bash
     - name: Fetch helm charts
       run: |
-        helm repo add jetstack https://charts.jetstack.io
-        helm repo update
```

Comment:
These lines lasted just a breath 😜

Reply:
I know 😆 I was so concerned with fixing what I'd broke on merge that I kinda panicked over "best placement"